src/ui/profile_preferences.ui

```diff
@@ -225,45 +225,60 @@
          <layout class="QGridLayout" name="gridLayout_groupBox_miscellaneous">
           <item row="0" column="0">
            <widget class="QCheckBox" name="enableDarkTheme">
-            <property name="text">
-             <string>Use a dark theme</string>
-            </property>
             <property name="toolTip">
              <string>&lt;p&gt;Changes your Mudlet Style to a Dark Fusion Style.&lt;/p&gt;</string>
             </property>
+            <property name="text">
+             <string>Use a dark theme</string>
+            </property>
            </widget>
           </item>
-          <item row="1" column="0">
+          <item row="2" column="0">
```

Comment:
Unrelated changes by Qt Creator - unfortunately just noise but it is not worth the effort to be fixing this every time.